gips/0081-indexing-payments.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
As per https://github.com/graphprotocol/graph-improvement-proposals/pull/30#issuecomment-2270979948, I would suggest renaming to "0081.md".

Reply:
Holding off until all comments are resolved so as to not confuse Github